detox/test/integration/e2e/flaky.test.js

```diff
@@ -13,12 +15,10 @@ describe('Stub1', () => {
     } finally {
       detox.trace.endSection('Asserting various texts');
     }
-    throw new Error(`I'm only here to make things interesting!`);
-  });
 
-  it.skip('Skipped test', async () => {
-    // Checking that skipped tests are also traced
+    if (session.testSessionIndex === 0) {
```

Comment:
This serves as an extra check for `testSessionIndex` logic. If it does not increment in the consequent `-R, --retries` test runner calls via CLI, this test is going to fail and get the integration test suite red.